src/openrct2-ui/windows/TitleCommandEditor.cpp

```diff
@@ -150,7 +149,7 @@ static int32_t get_command_info_index(int32_t index)
 {
     for (int32_t i = 0; i < static_cast<int32_t>(NUM_COMMANDS); i++)
     {
-        if (_window_title_command_editor_orders[i].command == index)
```

Comment:
Index on line 148 should be the TitleScript type. Also rename the variable name to command.